aten/src/ATen/native/nested/cuda/NestedTensorTransformerFunctions.cpp

```diff
@@ -13,9 +13,7 @@
 // TODO Consider moving all flash_attention code, nested tensor included to
 // Transformer library
 
-#ifdef USE_FLASH_ATTENTION
 #include <ATen/native/transformers/cuda/flash_attn/fmha_api.h>
-#endif
```

Comment:
Again, I propose we keep it as is (otherwise it'll be a hell to fix internal builds)